Tests/UnitTests/SolanaPricesAPIsUnitTests/CoinGeckoPricesAPITests.swift

```diff
@@ -0,0 +1,49 @@
+import XCTest
+import SolanaPricesAPIs
+
+
+class CoinGeckoPricesAPITests: XCTestCase {
+    
+    let api = CoinGeckoPricesAPI(pricesNetworkManager: CoinGeckoMockNetworkManager())
+
+    func testGetCoinPrices() async throws {
+        let prices = try await api.getCurrentPrices(coins: ["BTC", "ETH"], toFiat: "USD")
+        XCTAssertEqual(prices["ETH"]??.value, 1055.31)
+        XCTAssertEqual(prices["BTC"]??.value, 19725.28)
```

Comment:
Needs to retrieve price by using coingecko id

Reply:
Done